kmath-stat/src/commonMain/kotlin/space/kscience/kmath/distributions/NormalDistribution.kt

```diff
@@ -14,14 +14,9 @@ import space.kscience.kmath.stat.RandomGenerator
 import kotlin.math.*
 
 /**
- * Implements [UnivariateDistribution] for the normal (gaussian) distribution.
+ * Implements [Distribution1D] for the normal (gaussian) distribution.
  */
-public class NormalDistribution(public val sampler: GaussianSampler) : UnivariateDistribution<Double> {
-    public constructor(
-        mean: Double,
-        standardDeviation: Double,
-        normalized: NormalizedGaussianSampler = ZigguratNormalizedGaussianSampler,
-    ) : this(GaussianSampler(mean, standardDeviation, normalized))
```

Comment:
I really don't understand the benefit of a top-level function instead of the secondary constructor here.

Reply:
It is just a question of code style. In my current state of mind, I believe that secondary constructors are just bad practice.